<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Empleados</class>
 <widget class="QWidget" name="Empleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>509</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>628</width>
    <height>509</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/fondo/imagen-llamativa-cintas-sobre-fondo-oscuro-generacion-ai_724548-22165.jpg);</string>
  </property>
  <widget class="QTableView" name="tblEmpleados">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>461</width>
     <height>401</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tablaEditable">
    <item alignment="Qt::AlignLeft|Qt::AlignTop">
     <widget class="QLabel" name="lblCedula">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Cedula</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCed">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblNombre">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtNombre">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblApellidos">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Apellidos</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight|Qt::AlignTop">
     <widget class="QLineEdit" name="txtApellidos">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
background-color: rgb(139, 69, 0);

}

QLineEdit:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;

border:2px solid rgb(120,0,171);
 background-color: rgb(139, 69, 0);
}

QLineEdit:focus{
background-color: rgb(139, 69, 0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}

</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblTelefono">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtTelefono">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblDireccion">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Direccion</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtDireccion">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblPuesto">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Puesto</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtPuesto">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblFecha">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Fecha ingreso</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtFecha">
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="lblJefatura">
      <property name="styleSheet">
       <string notr="true">QLabel{	
color:rgb(255,255,254);
font: 12pt&quot;Arial&quot;;
background-color: rgb(170, 85, 0);;
}</string>
      </property>
      <property name="text">
       <string>Jefatura SI // NO</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="QLineEdit" name="txtJefatura">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>134</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
/*background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid rgb(255,255,255);*/
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);


}

QLineEdit:hover{
/*background-color:rgba(0,0,0);
background-color: :rgba(12,12,12);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);*/

border-radius:10px;
font:12pt&quot;Arial&quot;;
/*color: rgb(255,255,255);*/
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}

QLineEdit:focus{
background-color:rgba(0,0,0);
border-radius:10px;
font:12pt&quot;Arial&quot;;
border:2px solid rgb(120,0,171);
}
</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>161</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="btnEliminar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}</string>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>420</y>
     <width>161</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="btnGuardar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}</string>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEditar">
      <property name="styleSheet">
       <string notr="true">QPushButton{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color:rgb(255,255,254);
border:2px solid #ffffff;
}
QPushButton:hover{
border-radius:10px;
font:12pt&quot;Arial&quot;;
color: rgb(255,255,255);
border:2px solid rgb(120,0,171);
  background-color: rgb(170, 85, 0);
}</string>
      </property>
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="recursos/images.qrc"/>
 </resources>
 <connections/>
</ui>
